README.md

```diff
@@ -1,2 +1,26 @@
-# random
-Random Number
+# Random Number Generator
+
+You are provided this flip function:
+```javascript
+function flip() {
+    returnMath.random() >=0.5;
+}
+```
+You must implement a randomNumber(n) function that generates a random number greater than or equal to 0, and less than input n.
+
+Constraints:
```

Comment:
how do you feel about establishing expectations/evaluation criteria so that they at least have a baseline and get an idea of what will be expected in the first place? Something around the lines of:

- code organization & formatting
- efficiency & solution approach
- commit comment clearness
- code documentation

thoughts?

Reply:
While I like this idea, I think we should leave it out for now since the idea is for them to have freedom to try to solve the problem, what do you think?

Reply:
I think it won't hurt to leave it as it is now, but I also think it will help us both the candidate and us to get better results overall. I'll mark as resolved in the mean time.